include/ibamr/IBFEMethod.h

```diff
@@ -781,6 +782,11 @@ class IBFEMethod : public IBStrategy
     SAMRAI::tbox::Pointer<SAMRAI::mesh::GriddingAlgorithm<NDIM> > d_gridding_alg;
     bool d_is_initialized = false;
 
+    /*
+     * Scratch data caching objects.
```

Comment:
Could you add some more comments explaining that these are used by different sets of `FEDataManager`s?

Reply:
Sure.